packages/instrumenter/test/unit/mutators/assignment-operator-mutator.spec.ts

```diff
@@ -72,4 +94,47 @@ describe(sut.name, () => {
     expectJSMutation(sut, 'a ||= `b`', 'a &&= `b`');
     expectJSMutation(sut, 'a ??= `b`', 'a &&= `b`');
   });
+
+  it('should only mutate what is defined in the mutator level', () => {
+    expectJSMutationWithLevel(
+      sut,
+      assignmentOperatorLevel.AssignmentOperator,
+      'a += b; a -= b; a *= b; a /= b; a <<= b; a &&= b;',
+      'a += b; a += b; a *= b; a /= b; a <<= b; a &&= b;',
+      'a += b; a -= b; a *= b; a /= b; a >>= b; a &&= b;',
+      'a += b; a -= b; a *= b; a /= b; a <<= b; a ||= b;',
+    );
+  });
+
+  it('should not mutate anything if there are no values in the mutation level', () => {
+    expectJSMutationWithLevel(sut, [], 'a += b; a -= b; a *= b; a /= b; a <<= b; a &&= b;');
+  });
+
+  it('should mutate everything if everything is in the mutation level', () => {
+    expectJSMutationWithLevel(
+      sut,
+      assignmentOperatorAllLevel.BooleanLiteral,
+      'a += b; a -= b; a *= b; a /= b; a <<= b; a &&= b;',
+      'a -= b; a -= b; a *= b; a /= b; a <<= b; a &&= b;',
+      'a += b; a += b; a *= b; a /= b; a <<= b; a &&= b;',
+      'a += b; a -= b; a /= b; a /= b; a <<= b; a &&= b;',
+      'a += b; a -= b; a *= b; a *= b; a <<= b; a &&= b;',
+      'a += b; a -= b; a *= b; a /= b; a >>= b; a &&= b;',
+      'a += b; a -= b; a *= b; a /= b; a <<= b; a ||= b;',
```

Comment:
It would be nice to write some comments at the end of each line, to see which mutation is being tested at a glance (e.g. `// mutated -`. 

Something else to keep in mind is that we will also need to ensure some code consistency between the tests. In some places only the sub-operators are defined, in this place the whole level is defined. And there are also a different number of tests for most mutators. I'll create an issue for this.

Reply:
Personally I almost always do four:
- empty level
- undefined level
- level with everything
- level with some, not all